### PR TITLE
Fix header file includes

### PR DESCRIPTION
**Pull request by OrkoHunter**
Use quotes instead of angular braces to import local header files.

### CODE REVIEW

**Comment by OrkoHunter** [comment]
I haven't done the `math.h` bit here. I prefer doing it in a separate PR because it will need a commit reversed.

File: src/lemon/cplex.cc [hunk 1]
```diff
@@ -20,10 +20,10 @@
 #include <vector>
 #include <cstring>
 
-#include <lemon/cplex.h>
+#include "lemon/cplex.h"
 
 extern "C" {
-#include <ilcplex/cplex.h>
+#include "ilcplex/cplex.h"
```
Comment (ysitu) [inline src/lemon/cplex.cc]: Do not change this.

File: src/lemon/graph_to_eps.h [hunk 2]
```diff
@@ -19,26 +19,26 @@
...
-#include<iostream>
-#include<fstream>
-#include<sstream>
-#include<algorithm>
-#include<vector>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <algorithm>
+#include <vector>
```
Comment (ysitu) [inline src/lemon/graph_to_eps.h]: Avoid unnecessary changes.

File: src/lemon/cplex.cc (hunk 1 above)
Comment (OrkoHunter) [inline src/lemon/cplex.cc]: Yeah got it.

File: src/lemon/graph_to_eps.h (hunk 2 above)
Comment (OrkoHunter) [inline src/lemon/graph_to_eps.h]: Removed this change.

File: src/lemon/clp.cc [hunk 3]
```diff
@@ -16,8 +16,8 @@
  *
  */
 
-#include <lemon/clp.h>
-#include <coin/ClpSimplex.hpp>
+#include "lemon/clp.h"
+#include "coin/ClpSimplex.hpp"
```
Comment (ysitu) [inline src/lemon/clp.cc]: Revert line 20.

File: src/lemon/lmath.h [hunk 4]
```diff
@@ -27,7 +27,7 @@
 ///
 ///This file includes the standard math library (cmath).
 
-#include<cmath>
+#include <cmath>
```
Comment (ysitu) [inline src/lemon/lmath.h]: Revert this file.

**Comment by OrkoHunter** [comment]
Can we have this and #8 in now?
